### PR TITLE
Ports Humans from TG - Soul Massacre 

### PR DESCRIPTION
**Pull request by Alecksohs**
## What Does This PR Do

Ports TG's Human Sprites to Paradise

## Why It's Good For The Game

The human sprites are shit. These are less shit. What greater justification do you need? What you want an essay? Okay.

Paradise's Human Sprites are very very old. Their age shows, as they are closer to Yellow Goblins than anything human. TGStation takes the silhouette and implements hue shifting, shading, and implication into the sprite, allowing it to be compatible with all clothes yet look better.  I don't wanna write an essay thats what you get I'm tired

Explanation of Preset Skin Tones:
Original in Human Words:
If the bodyflags of the a species of a mob with dna and  the flag is active called has_skin_tone (humans do)
take a value of -50 - 50
If skintone is greaterthan or equalto 0, add numbers to every pixel based on the skintone_number. 
If its not subtract this number from the pixel. Each pixel has an RGB value, or 3 different numbers from 0-255
Therefore its literally a slope to the color black, not skin tones.

This is bad. This will never get you anywhere. You just get darker sprites, which isn't how skin works. It isn't how math works. Nobody colors like this. Vulpakinian use Multiplication with RGB, which is better but not good still. Outlines get overblown and hueshift? Whats that? Doesn't work. Solution? Use premade sprites inwhich have good sprite principles, can be updated individually and look nice together. 

Oh and I also gave you Classic, a skintone in which is just 35 skin tone (what most people use) of the old sprite
You're welcome

## Images of changes

UPDATED

image

image
image
image
image
image
image
image
image
image
image
image
image
image
image

## Testing

I loaded it into the game. I played with the skintones and they work for the shitcode we employ, still its better though. Everything works fine.

## Changelog
:cl:
tweak: Ported TG Humans
tweak: Updated Cardboard Cutouts for TG Human port
/:cl:

### CODE REVIEW

**Comment by A-Medster** [comment]
They feel so..
Fleshy..

**Review by JimKil3 — APPROVED** [review]
lgtm :) can't wait to get emails for all the discussion on this

**Review by Contrabang — APPROVED** [review]
code is fine

**Comment by Alecksohs** [comment]
I did it I saved the server

**Review by Henri215 — CHANGES_REQUESTED** [review]
Some cardboards from ```icons/obj/cardboard_cutout.dmi``` will have to be updated now that humans look different.

**Review by DGamerL — APPROVED** [review]
The two sides of reviewers
image

**Comment by Alecksohs** [comment]
image
WAIT IS THIS ONE ALREADY USING TG SPRITES LMFAO

**Comment by Alecksohs** [comment]
image
Fixed

**Comment by Alecksohs** [comment]
image
Simplemobs fixed

**Review by Henri215 — APPROVED** [review]
Its looking good to me now.

**Comment by Mitchs98** [comment]
These just look extremely off to me, very changeling flesh suit-y but less bloated. They just look fake, too shiny and smooth looking, while also being kinda wrinkly and fat roll-y in certain spots. What we have isn't exactly the peak in spriting but it looks better to this to me. 

It could be a coding issue, it could not be. I have no idea. But this just looks like a downgrade in its current state regardless of how bad our current sprites may or may not be.

**Comment by Alecksohs** [comment]
> These just look extremely off to me, very changeling flesh suit-y but less bloated. They just look fake, too shiny and smooth looking, while also being kinda wrinkly and fat roll-y in certain spots. What we have isn't exactly the peak in spriting but it looks better to this to me.
> 
> It could be a coding issue, it could not be. I have no idea. But this just looks like a downgrade in its current state regardless of how bad our current sprites may or may not be.

As stated in the discord, it could be a contrast issue and likely is. I'd request assistance from any coder who would like to assist, create a human skin preset system with skin colors having their own sprites. If said coder would like to be a baller, he could just port the skin tone code itself. However my coding knowledge isn't that far.

**Comment by Alecksohs** [comment]
Working on the skincode now, also fixing a few gripes with the TG sprite itself

**Comment by Alecksohs** [comment]
image
image
image
image
image

Using preset skintones, here are the ones I've created.

**Comment by Alecksohs** [comment]
image
the classic tm
image

**Comment by Alecksohs** [comment]
I'm ruining vox too but thats a separate pr

**Comment by Mitchs98** [comment]
Looks quite a bit better than the original implementation of the port, by far.

**Comment by Alecksohs** [comment]
This is quite feature complete, now.

File: code/modules/mob/living/carbon/human/species/human_species.dm
Comment (S34NW) [inline code/modules/mob/living/carbon/human/species/human_species.dm:1]: You've changed this file to CRLF instead of our standard of LF. CI failing.

Comment (Alecksohs) [reply code/modules/mob/living/carbon/human/species/human_species.dm:1]: I used the DM editor, strange. How do I fix that?

Comment (Alecksohs) [reply code/modules/mob/living/carbon/human/species/human_species.dm:1]: I imported it into notepad++ changed it to UnixLF and then saved that as a dm and just pushed it.

**Comment by DT2399** [comment]
the new sprites looks so good

**Comment by TerraTheUnwise** [comment]
please don't

**Review by S34NW — APPROVED** [review]

**Comment by Alecksohs** [comment]
I forgot to fix the butt pixels chat on the chestnut skin tone this is my fault please put down your pitchforks idk how to put out the fire (fix it)

**Pull request merged**